core/storage/storage_models_test.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for Oppia storage models."""
+import inspect
+
+from core.platform import models
+from core.tests import test_utils
+
+
+class StorageModelsTest(test_utils.GenericTestBase):
+    """Tests for Oppia storage models."""
+
+    def test_all_model_names_unique(self):
+        all_model_names = []
+        for name in models.NAMES.__dict__:
```

Comment:
In general, we should avoid using double-underscore properties. What is stopping you from simply enumerating models.NAMES?

Reply:
models.NAMES is an enum which cannot be iterated. So I had to work around it this way. another thing I could do is to do vars(models.NAMES). Should I do that?

Reply:
Ok Will do.

Reply:
Done